contrib/ceph/micro-osd.sh

```diff
@@ -0,0 +1,90 @@
+#
+#    Copyright (C) 2013,2014 Loic Dachary <loic@dachary.org>
```

Comment:
@dmp42 @stevvooe @noahdesu At worst we could do as done with gvm-installer, wget from the go-ceph repo?